dipy/reconst/tests/test_ivim.py

```diff
@@ -74,7 +74,10 @@ def setup_module():
                             120., 140., 160., 180., 200., 300., 400.,
                             500., 600., 700., 800., 900., 1000.])
 
-    gtab_no_b0 = gradient_table(bvals_no_b0, bvecs.T, b0_threshold=0)
+    # Avoid raising a warning here due to the lowest bval and the provided b0
+    # threshold
```

Comment:
Can you remove this comment?  It is not necessary.

Reply:
Removed in 28ae655.